enterprise-beans/4.0/_index.md

```diff
@@ -7,18 +7,25 @@ summary: "Release for Jakarta EE 9"
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record]()
-  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
+  * [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/release-plan/4.0-PLAN.adoc)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (PDF)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (HTML)
 * [Jakarta Enterprise Beans 4.0 Javadoc](./apidocs)
 * [Jakarta Enterprise Beans 4.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
   * [jakarta.ejb:jakarta.ejb-api:jar:4.0.0]()
 
+The Jakarta Enterprise Beans 4.0 intended scope includes all plans detailed in the link:https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan[Jakarta EE 9 Release Plan], which in essence involves a backwards incompatible namespace change from `javax.ejb` to `jakarta.ejb`.  In addition the Jakarta Enterprise Beans 4.0 Release Plan will involve three changes:
+
+ - Removal of methods relying on `java.security.Identity`
+ - Removal of Support for Distributed Interoperability
+ - Mark optional EJB 2.x API Group
```

Comment:
Tracy Burroughs just pointed out to me that the pruning of XML RPC also affects the EJB API.  He's making more thorough comments on your actual EJB Plan PR.

Reply:
Updated the description in the PR to include those.  Plan updated in this PR https://github.com/eclipse-ee4j/ejb-api/pull/65